src/components/ADempiere/Favorites/index.vue

```diff
@@ -61,28 +59,10 @@ export default {
   },
   methods: {
     getFavoritesList() {
```

Comment:
Hi, tis is a bad form of resolve this, the search should be on dashboard instead

Reply:
Hi, changed library side data conversion for dashboards and added missing attributes and fixed attribute errors.

https://github.com/erpcya/gRPC-Data-Client/pull/28